<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chat</class>
 <widget class="QMainWindow" name="Chat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="sendBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="sendBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>146</y>
      <width>401</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="recBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>6</y>
      <width>401</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="friendList">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>81</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="closed">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>230</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>好   友   栏</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addFriendBtn">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>250</y>
      <width>181</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>添加好友</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>220</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>好友账号：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="addFriendE">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>220</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="inviteBtn">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>0</y>
      <width>61</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string>邀请好友来</string>
    </property>
   </widget>
   <widget class="QLabel" name="onlineNumberLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>在线人数：</string>
    </property>
   </widget>
   <widget class="QLabel" name="onlineNumber">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>不知道捏qwq</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
